modules/refme.adoc

```diff
@@ -0,0 +1,4 @@
+[id="hello"]
```

Comment:
Missing context, which is a problem.

Reply:
@vikram-redhat, oh, yeah, but was useful to confirm that injecting a relative path attribute does produce valid links locally regardless of where an assembly lives.